Tools/WebKitTestRunner/TestInvocation.cpp

```diff
@@ -176,8 +178,10 @@ void TestInvocation::invoke()
 
     // FIXME: We should clear out visited links here.
```

Comment:
This can probably be removed since we call WKPageSetAddsVisitedLinks(page, false).  We just don't test visited links in WebKitTestRunner, which has been fine for the last decade.